docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -66,296 +97,193 @@ image::images/lens_end_to_end_1_3.png[Changed type and dropped clientip field]
 [role="screenshot"]
 image::images/lens_end_to_end_1_4.png[Flyout config open]
 
-. Click *Save and return*.
-
-[discrete]
-[[view-the-distribution-of-visitors-by-operating-system]]
-=== View the distribution of visitors by operating system
+. Click *Save and return*. You will return to the dashboard, where you can customize
```

Comment:
```suggestion
. Click *Save and return*. 

. On the dashboard, drag the bottom corner of the panel until the panel takes up one quarter of the dashboard so that you can display an additional four metrics.
```